src/main/java/seedu/duke/TravelActivityList.java

```diff
@@ -65,4 +65,27 @@ public String getDescription(String plan){
         }
         return "cant be found";
     }
+
+    public void searchTask (String taskName) {

```

Comment:
I think you can use array list to simplify this method. Similar to getDescription